### ISSUE TITLE
Décoration du Fabvélo

### ISSUE THREAD

**Issue by milenasonneveld** [openfab-lab/FabMob#14]
Finalement on peut être jaloux du truck de BeMaker. Ils on refait la déco. Maintentant à nous de faire mieux ;)

![img_2818](https://user-images.githubusercontent.com/31619215/45230022-d6655000-b2c7-11e8-8c8d-bf68542a5be0.JPG)

On veut des stickers sur un support en bois qui vient dans le cadre du vélo cargo.
![bullittbillboard](https://user-images.githubusercontent.com/31619215/45230181-60151d80-b2c8-11e8-8191-b93837a9673f.jpg)

### Process
En faire un job laser pour le support
En faire un job design pour le stickers

### FilesResources
[eBullitt Billboard Template .pdf](https://github.com/openfab-lab/FabMob/files/2361702/eBullitt.Billboard.Template.pdf)

**Comment by nicolasdb** [comment]
Trop la classe, 
@DewiBrunet c'est pas l'occasion de jouer avec ta machine à découpe vynil? 

Comment est-ce qu'on définit plus facilement une demande de job dans la communauté? On transmet à @zuperninja  ou on mettrait pas en place un template de cahier des charges pour mieux définir le job?

**Comment by DewiBrunet** [comment]
Yes effectivement c'est dans mon agenda de me former sur ma découpeuse vinyle et ça peux être un bon exercice pratique :) (par contre je suis limité à du 30cm en largeur...)

@milenasonneveld tu sais me donner plus de précision niveau taille, couleurs, typo et contenu des stickers?

**Comment by milenasonneveld** [comment]
https://www.dropbox.com/sh/lobqv4aw0e6gx53/AADWAqVkRetkclqiNpOSo4Dta?dl=0
Pour une première proposition (sachant que le logo de Vélo M² va encore changer).

Les deux matériaux intéressants à l'Openfab :
![img_3506](https://user-images.githubusercontent.com/31619215/48639699-aa0b3580-e9d4-11e8-865f-0ccbafc9fe17.jpg)
![img_3507](https://user-images.githubusercontent.com/31619215/48639701-aa0b3580-e9d4-11e8-9486-298f54ab7b45.jpg)

**Comment by nicolasdb** [comment]
le polypropylène en photo, c'est idéal pour les pochoirs. 
Ou pour garder quelque chose de souple, c'est bien résistant. 

Pour le bulitt, j'irai pour un multiplex clair avec gravure foncée et verni (à l'eau). ça irai en accord avec les modules. >swag

**Comment by nicolasdb** [comment]
si on voit que ça ne tient pas bien, on pourrait couper le template à la cnc en trespa 4mm. C'est blanc en surface et on peut le graver au laser (noir)
https://nicolasdb.github.io/post/cnctrespa/
